.github/workflows/publish-storybook.yml

```diff
@@ -1,15 +1,15 @@
 name: Build storybook & deploy
-on:
-  push:
-    branches:
-      - master
+on: push
+  # push:
+  #   branches:
+  #     - master
```

Comment:
We've gotta find a better way to test these things. When making changes to the GH action workflow, this makes enough sense, but it doesn't look like that's what you're doing here.

Reply:
I agree. I'm going to look into chromatic again.